netci.groovy

```diff
@@ -1897,7 +1897,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                             if (enableCorefxTesting) {
                                 def fxRoot = "%WORKSPACE%\\_\\fx"
```

Comment:
Err, sorry, the one below ($WORKSPACE%) appears definitely horked.  I'm not sure what's wrong with this one, but it's hard to believe we can't pass the right thing via the command line.